### DISCUSSION TITLE
Export to Sqlite

### DISCUSSION THREAD

**Discussion by eadmaster** [dbeaver/dbeaver#20063] [Q&A] [answered]
is it possible to export table data into a sqlite db?

(should preserve the native db datatypes)

https://dbeaver.com/docs/wiki/Data-transfer/

**Comment by LonwoLonwo** [comment, marked as answer]
Yes, you can export tables in the SQLite database. 
If this database is not read-only.
https://dbeaver.com/docs/wiki/Data-migration/

**Comment by eadmaster** [comment]
i do not see the option

(SQL INSERT Statement != sqlite)

**Reply by m-tay** [reply]
Export Data > Database > Pick a SQLite DB that you have previously added as a connection

(Mostly for future readers who come across this when googling!)
